thoth/user_api/api_v1.py

```diff
@@ -700,14 +730,18 @@ def schedule_kebechet_webhook(body: typing.Dict[str, typing.Any]):
     # Not schedule workload if pre-processed payload is None.
     if preprocess_payload is None:
         return
+
+    # Should we share a kebechet cache here?
```

Comment:
What cache do you mean? Here and above?

Reply:
Nvm, I don't think this would work because the odds that we get an identical payload is tiny even if the end result is the same